src/pcsswebservice/src/main/java/ca/bc/gov/open/pcss/pcsswebservice/WebServiceConfig.java

```diff
@@ -31,31 +30,55 @@ public DispatcherServletPath dispatcherServletPathProvider() {
     }
 
     @Bean(name=Bus.DEFAULT_BUS_ID)
-    public SpringBus springBus(LoggingFeature loggingFeature) {
+    public SpringBus springBus() {
 
         SpringBus cxfBus = new SpringBus();
-        cxfBus.getFeatures().add(loggingFeature);
-
         return cxfBus;
     }
 
+    @Bean
+    public Endpoint CivilEndpoint(Bus bus, PcssCivilEndpoint pcssCivilEndpoint) {
+
+        EndpointImpl endpoint = new EndpointImpl(bus, pcssCivilEndpoint);
+        endpoint.publish("/pcssCivil");
+
+        return endpoint;
+    }
 
     @Bean
-    public LoggingFeature loggingFeature() {
+    public Endpoint CommonEndpoint(Bus bus, PcssCommonEndpoint pcssCommonEndpoint) {
 
-        LoggingFeature loggingFeature = new LoggingFeature();
-        loggingFeature.setPrettyLogging(true);
+        EndpointImpl endpoint = new EndpointImpl(bus, pcssCommonEndpoint);
+        endpoint.publish("/pcssCommon");
```

Comment:
Can we get the name from `webMethods: pacakge.whatever:serviceWs`

Reply:
And done!